### ISSUE TITLE
TestDownloadOnly/group/ExpectedDefaultDriver: got driver "none", expected "kvm2"

### ISSUE THREAD

**Issue by tstromberg** [kubernetes/minikube#7087]
As seen at https://storage.googleapis.com/minikube-builds/logs/7075/ce5bf6c/none_Linux.html#fail_TestDownloadOnly%2fgroup%2fExpectedDefaultDriver

```
--- FAIL: TestDownloadOnly/group/ExpectedDefaultDriver (0.09s)
aaa_download_only_test.go:125: (dbg) Run:  out/minikube-linux-amd64 profile list --output json
aaa_download_only_test.go:143: got driver "none", expected "kvm2"
start output: -- stdout --
```

Assuming the machine the test ran on does not support kvm2, perhaps we should just delete this test.

**Comment by prasadkatti** [comment]
Shouldn't this issue get closed with the above PR being merged? The PR says "fixes #7087".

**Comment by tstromberg** [comment]
@prasadkatti - Yes. Thank you for noticing!
